ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py

```diff
@@ -0,0 +1,126 @@
+#
+# Copyright 2022 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import print_function
+from datetime import datetime
+from ecchronoslib import table_formatter
+
+
+def print_verbose_repair_job(repair_job, max_lines, full=False):
```

Comment:
"verbose_repair_job"? isn't this only viable for schedules? Also i don't like the "job" thingy